emuu-client/src/components/UserProfile/Profile.js

```diff
@@ -1,9 +1,90 @@
-import React from "react";
 import "./Profile.scss";
 import Feeds from "./Feeds";
 import UserInfo from "./UserInfo";
+import React, { useState , useEffect } from "react";
+import "../../Firebase.js";
+import {storage, db} from "../../Firebase.js";
+import { ref,getStorage, uploadBytes, getDownloadURL } from "firebase/storage";
+import AddIcon from '@mui/icons-material/Add';
+import { collection, getDocs } from "firebase/firestore";
 
 function Profile() {
+
+
+  // Store uploaded file
+  //const [file, setFile] = useState("");
+
+
+  //File upload
+  //function handleChange(event) {
+  //  setFile(event.target.files[0]);
+  //}
+  //If a user doesn't choose a file and tries to upload, error will appear
+  //const handleUpload = () => {
+//   if (!file) {
+//      alert("Please upload an image first");
+//    }
+//
+//    //Store into video folder in firebase storage
+//    const storageRef = ref(storage, `/images/${file.name}`);
+//
+//    //Upload to firebase function
+//    const uploadTask = uploadBytesResumable(storageRef, file);
+//
+//    uploadTask.on(
+//      "state_changed",
+//      (snapshot) => {},
+//      (err) => console.log(err),
+//      () => {
+//        // download url
+//        getDownloadURL(uploadTask.snapshot.ref).then((url) => {
+//          console.log(url);
+//        });
+//      }
+//    );
+//  };
+    useEffect(()=>{console.log(JSON.parse(localStorage.getItem("user")));getDocs(collection(db, "Users")).then(data =>{
```

Comment:
What exactly does this line of code do? And all of this is stored as console data.